pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths =
+    tests
```

Comment:
Do we need this file? If so why?:D

Reply:
It is just so that we can run
```
python3 -m pytest
```
instead of 
```
python3 -m pytest tests
```
Currently, if we do the former, then pytest will also discover the tests in the coockiecutter folder and try to run them